cmd/main.go

```diff
@@ -32,14 +32,32 @@ func Run(name string, args []string) error {
 	opts.args = flags.Args()
 	setupLogging(opts)
 
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	signalChan := make(chan os.Signal, 1)
+	signal.Notify(signalChan, os.Interrupt)
```

Comment:
I know you are moving code, but you could consider using signal.NotifyContext that would simplify this code

Reply:
Nice, I took this way